volmdlr/edges.py

```diff
@@ -2749,7 +2749,46 @@ def to_2d(self, plane_origin, x1, x2):
 
     def polygon_points(self):
         return self.points
+
+    def unit_direction_vector(self,abscissa=0.):
+        l = self.length()
+        if abscissa >= l :
+            abscissa2 = l
+            abscissa = abscissa2 - 0.001*l
+            
+        else :
+            abscissa2 = abscissa + 0.001*l
+            if abscissa2 > l:
+                abscissa2 = l
+            
+        tangent = self.point_at_abscissa(abscissa2)-self.point_at_abscissa(abscissa)
+        tangent.normalize()
+        return tangent
     
+    def unit_normal_vector(self, abscissa):
+        return None
+    
+    def minimum_curvature_radius_at_u(self, u: float):
```

Comment:
Better to return curvature=1/radius rather than radius because radius can be inf
Rename to minimum_curvature (u is obvious on curves)